azerothcore/azerothcore-wotlk
# fix (db/quest) Add missing objective marker quest 9454

## Changes Proposed:
-  Add missing objective marker quest 9454

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/2838

## SOURCE:


## Tests Performed:

- Builds
- Performs


## How to Test the Changes:


1. be alliance
2. .quest add 9454
3. .go creature 61469
4. observe it now has quest tracker objective when you highlight the creature when u have the quest
![image](https://user-images.githubusercontent.com/16887899/150443099-beb87d90-72aa-4b76-a512-8cb90d79088c.png)
5. when you abandon the quest the objective no longer shows as intended
![image](https://user-images.githubusercontent.com/16887899/150443127-ee21f749-c506-421e-accc-1b4e7a7018db.png)


## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
